scss/core/_variables.scss

```diff
@@ -472,11 +472,35 @@ $border-radius-sm:            .25rem !default;
 
 $rounded-pill:                50rem !default;
 
-$level-1-box-shadow:          0 .0625rem .125rem rgba(0, 0, 0, .15), 0 .0625rem .25rem rgba(0, 0, 0, .15) !default;
-$level-2-box-shadow:          0 .125rem .25rem rgba(0, 0, 0, .15), 0 .125rem .5rem rgba(0, 0, 0, .15) !default;
-$level-3-box-shadow:          0 0 .625rem rgba(0, 0, 0, .15), 0 0 1rem rgba(0, 0, 0, .15) !default;
-$level-4-box-shadow:          0 .625rem 1.25rem rgba(0, 0, 0, .15), 0 .5rem 1.25rem rgba(0, 0, 0, .15) !default;
-$level-5-box-shadow:          0 1.25px 2.5rem rgba(0, 0, 0, .15), 0 .5rem 2.5rem rgba(0, 0, 0, .15) !default;
```

Comment:
[curious] If we rename/remove these variables, this will have to be treated as a breaking change, and consumers will need to migrate to the new `box-shadow-*` variables instead as part of the upgrade. Would it make sense to have this change be additive, rather than removing these existing variables to avoid a breaking change?

Reply:
You're right, we should leave these variable as is and just add new ones, thanks!